test/api/access_codes/delete.test.ts

```diff
@@ -0,0 +1,46 @@
+import test, { type ExecutionContext } from "ava"
+
+import { getTestServer } from "fixtures/get-test-server.ts"
+
+test("POST /access_codes/update", async (t: ExecutionContext) => {
+  const { axios, seed, db } = await getTestServer(t)
+  const device_id = seed.ws2.device1_id
+
+  const {
+    data: { access_code },
+  } = await axios.post(
+    "/access_codes/create",
+    {
+      device_id,
+      name: "Test Access Code",
+      code: "1234",
+    },
+    {
+      headers: {
+        Authorization: `Bearer ${seed.ws2.cst}`,
+      },
+    }
+  )
+
+  t.is(access_code.code, "1234")
+
+  const res = await axios.post(
```

Comment:
```suggestion
  const res = await axios.delete(
```

Reply:
Aren't we doing `post` for everything?

Reply:
No, but I realize this is confusing. POST is supported for all endpoints, but this is just a convenience. The SDK will always use the more semantic method (and we will use that in the docs as well).

Reply:
gotcha thanks for clarifying